src/utils/project-content/build-project-content.ts

```diff
@@ -9,9 +9,97 @@ import getProjectContent from './get-project-content';
 import { getProjectExports } from './get-project-exports';
 import writeProjectContent from './write-project-content';
 
+const nodeModulesBinPath = path.normalize(
+  path.resolve(__dirname, '../../../node_modules/.bin')
+);
+
 export const buildProjectContent = async (
   projectContent: IProject
 ): Promise<IProject> => {
+  return await applySASS(await applyNPMBundler(projectContent));
+};
+
+async function applySASS(projectContent: IProject): Promise<IProject> {
+  const tmpDir = createTemporaryDirectory();
+  await writeProjectContent(tmpDir.name, projectContent);
+  const builtProjectContent = getProjectContent(tmpDir.name);
+  const sassBinaryPath = path.resolve(nodeModulesBinPath, 'sass');
+
+  for (const collection of builtProjectContent.collections) {
+    for (const fragment of collection.fragments) {
+      const cssPath = fragment.metadata.cssPath.toLowerCase();
+
+      if (cssPath.endsWith('.scss') || cssPath.endsWith('.sass')) {
+        const absoluteFragmentPath = path.resolve(
+          builtProjectContent.basePath,
+          'src',
+          collection.slug,
+          fragment.slug
+        );
+
+        const absoluteCSSPath = path.resolve(
+          absoluteFragmentPath,
+          fragment.metadata.cssPath
+        );
+
+        const absoluteCSSDestinationPath = absoluteCSSPath.replace(
+          /\.s(a|c)ss$/,
```

Comment:
Just curiosity, don't change anything. `(` is usually used for capturing groups right? is this the same as `/\.s[ac]ss$/`